KenticoListApp/TodoList.Api/Controllers/ItemsController.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Web.Http;
+using System.Web.Http.Controllers;
+using TodoList.Api.Models;
+
+namespace TodoList.Api.Controllers
+{
+    public class ItemsController : ApiController

```

Comment:
It is called `ItemsController` although the model it is responsible for is called `ListItem`. Rename one of them to make it consistent!